server/config.ts

```diff
@@ -133,9 +133,6 @@ export default {
   },
   featureFlags: {
     addTestErrorEndpoint: flag('FEATURE_ADD_TEST_ERROR_ENDPOINT', false),
-    // Whether to hide the 'Days since last review'/'Days on level' columns
-    hideDaysColumnsInIncentivesTable: flag('FEATURE_HIDE_DAYS_COLUMNS_IN_INCENTIVES_TABLE', false),
```

Comment:
`FEATURE_HIDE_DAYS_COLUMNS_IN_INCENTIVES_TABLE` was only used in the old table, so i think it's correct to remove it.

it still appears in the various helm_deploy/values files and some .env files too so i'd get delete it there too